middleware/RetryHandler.md

```diff
@@ -10,7 +10,11 @@ Provide a reusuable component that provides application developers with effectiv
 - A `retry-attempt` header SHOULD be added which contains a numeric value representing the retry number.  This value will be used for diagnostics and determining the effectiveness of the retry handler.
 - Retries MUST respect the `retry-after` response header if provided.
 - Where no `retry-after` header is provided by the server, an exponential backoff with random offset hueristic should be used to determine the retry delay.
-- Retries should be limited to a maximum value.
+- Retries should be limited to a maximum delay value. The default value for this is set at 180 seconds.
+- The client code can specify a custom value for the maximum delay.
+- Retries will be based on the cumulative retry delay compared against the maximum delay specified by the client code.
```

Comment:
To clarify, can we state that this is cumulative across all retries stemming from a single call? This is applicable to both 429 as well as 503/504.

Reply:
Done